worldedit-core/src/main/java/com/sk89q/worldedit/util/TreeGenerator.java

```diff
@@ -108,6 +108,9 @@ public boolean generate(EditSession editSession, BlockVector3 pos) throws MaxCha
                 return editSession.getWorld().generateTree(this, editSession, pos.subtract(0, 1, 0));
             }
         },
+        MANGROVE("Mangrove tree", "mangrove"),
```

Comment:
This doesn't map these anywhere, so these won't do anything. They'll need to be mapped to tree features in the platforms

Reply:
Ah yeah mb, I was copying over from FAWE changes and we lazily just look up the enums via Bukkit's methods